pandas/core/frame.py

```diff
@@ -341,6 +348,24 @@
 ...
 ValueError: columns overlap but no suffix specified:
     Index(['value'], dtype='object')
+
+>>> df1 = pd.DataFrame({'left': ['foo', 'bar']})
```

Comment:
can you add an example of an inner and left merge here (and put them right before this)

Reply:
Done, could you please check if this is similar to what you have in mind?